XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
+Transactions are held on each side of the channel, in our case on the exchange’s XUD, representing the balance of each party. They use a multi-signature address as their input (the funding address) and point at two different addresses for their output, each controlled by one exchange. Setting up payment channels can be compared to transferring coins into a sidechain - coins have to be committed to a payment channel via a native on-chain transaction. This transaction 'locks' coins on the underlying chain and 'unlocks' them in the channel.
 
-Each participating exchange runs its own XUD instance, which opens payment channels on each chain it chooses to support. This means, if an exchange wants to enable its users to trade bitcoin and litecoin through Exchange Union, it has to open a payment channel on the litecoin blockchain AND on the bitcoin blockchain to at least one other exchange in the union. Payment channels in XU are meant to be kept open long-term. Also, the current design doesn't encourage high volume transactions in one single channel and most implementations currently set upper limits as a 'training wheels restriction' (e.g. lnd's max. channel size is: 0.16777216 BTC). To achieve scalability, we will rely on multiple channels, which carry out partial transfers of one atomic payment as proposed in the [new AMP protocol by Laolu and Conner Fromknecht](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-February/000993.html). This allows:
-- to elegantly get around channel volume restrictions
-- to elegantly increase/decrease volume when needed by simply opening/closing new channels
-- achieve better connectivity (direct channels) between different XUDs
+With this mechanism, payment channels are trustless. Trustless means one bitcoin in a payment channel is backed by one locked bitcoin on the blockchain and can be transferred between two parties securely, even though only the opening and closing transaction are included in the blockchain. Simplified, this is secure, because all transactions are valid on the underlying blockchain and can be broadcast by either party at any time. So both parties can rest assured to be in possession of the respective funds when receiving a transaction from the other party in a channel.
```

Comment:
I don't think it's accurate to say that anything is "backed" - the bitcoin in the payment channel *is* on the blockchain. Second sentence can just say "This is secure because..." And last sentence can say "Both parties are in possession of their respective funds within the payment channel."

Reply:
> I don't think it's accurate to say that anything is "backed" - the bitcoin in the payment channel is on the blockchain. 

Pegged?

> This is secure because...

Fixed.

> Both parties are in possession of their respective funds within the payment channel.

Fixed.